content/docs/18-knowledgebase/00-how-to/2-palette-sso-with-adfs.md

```diff
@@ -0,0 +1,222 @@
+* Spectro Team: leave at the default of `groups`
+
+![Enable Palette OIDC SSO](palette-sso-with-adfs-images/20-configure_palette_oidc.png)
+
+
+When all the information has been entered, click the **Enable** button at the button to enable SSO. You should receive an `OIDC configured successfully` message.
+
+
+## 3 - Create Teams in Palette
+
+All that's left to do now is to create Team(s) in Palette for the groups claim(s) that were configured in AD FS, and give them the appropriate permissions. For this example, we will create the `Admins` team and give it `Tenant Admin` permissions. You can repeat this for any other team that you've configured group claims for.
+
+Open a web browser and navigate to your Palette subscription. Navigate to **Tenant Settings** --> **Users & Teams** --> **Teams** tab. Click the **+ Create Team** button.
+
+![Create Palette Team](palette-sso-with-adfs-images/21-create_team.png)
+
+
+Specify `Admins` in the **Team name** field. We don't need to set any members now, as this will happen automatically from the SSO. Click **Confirm** to create the team.
+
+![Name Palette Team](palette-sso-with-adfs-images/22-name_team.png)
+
+
+You'll be returned back to the list of Teams. Click on the newly created `Admins` Team to open its details screen. To give this team administrative access to the entire tenant (and thus all Projects in it), we will give this Team the `Tenant Admin` role. Click on **Tenant Roles** --> **+ Add Tenant Role**:
+
+![Palette Tenant Roles](palette-sso-with-adfs-images/23-tenant_roles.png)
+
+
+Click on **Tenant Admin** to enable the role, this automatically enables all other roles with fewer permissions. Click **Confirm** to add the role:
+
+![Add Tenant Role](palette-sso-with-adfs-images/24-add_tenant_role.png)
+
+
+You will see the `Roles have been updated` message to confirm the change. Repeat this procedure for any other teams you need, ensuring they're given the appropriate permissions. Finally click the `X` next to **Team Details** in the top left corner to exit this screen.
+
+Congratulations! You have now successfully configured Palette SSO based on OIDC with Microsoft AD FS!
+
+
+# Validation
+Let's log in through SSO as a user that is a member of the `SpectroTeam - Admins` group in Active Directory to verify that we're automatically getting added to the `Admins` group in Palette.
+
+If you're still logged into Palette with a non-SSO user, log out first by clicking on your name at the top right and selecting `Logout from <tenant name>`:
+
+![User Logout](palette-sso-with-adfs-images/25-user_logout.png)
+
+
+Notice that the Palette login screen now only shows a big **Sign in** button and no longer a username/password field. Below this button there is an **SSO issues? --> Use your password** link. This can be used to bypass SSO and login with a local Palette account in case there's ever a problem with SSO and you need to access Palette without SSO.
```

Comment:
```suggestion
The Palette login screen now displays a **Sign in** button and no longer presents a username and password field. Below the **Sign In** button, there is an **SSO issues? --> Use your password** link. This link can be used to bypass SSO and log in with a local Palette account in case there is an issue with SSO and you need to access Palette without SSO.
```